### PR TITLE
Added Menu.jsx desktop version

### PR DESCRIPTION
**Pull request by BenZen59**

### CODE REVIEW

**Review by KeerhemFR — DISMISSED** [review]
I see no evil.
Just will have to change the css sheet into styled components to match with all the project.
And will have to switch menu from this to something with Links.

File: src/components/Menu/style.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  background-color: var(--mColor);
+}
+
+li {
+  float: left;
+}
+
+li a,
+.dropBtn {
+  display: inline-block;
```
Comment (Hacene010) [inline src/components/Menu/style.css]: a simple flex here is better I think

File: src/components/Menu/style.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: var(--mColor);
+}
+
+li {
+  float: left;
```
Comment (Hacene010) [inline src/components/Menu/style.css]: be aware aboute the float !!!!

File: src/components/Menu/index.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import './style.css';
+
+export default function Menu() {
+  return (
+    <div className="menu">
+      <div className="dropDown">
+        <div className="dropBtn">Glaucome</div>
+        <div className="dropDownContent">
```
Comment (Hacene010) [inline src/components/Menu/index.jsx]: Is this a button or div

File: src/components/Menu/style.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+.dropDownContent {
+  color: black;
+  padding-left: 10px;
+  padding-right: 10px;
+  display: none;
+  position: absolute;
+  background-color: #f9f9f9;
+  min-width: 160px;
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  z-index: 1;
```
Comment (Hacene010) [inline src/components/Menu/style.css]: That's not good to use this z-index

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Some evils to correct

File: src/components/Menu/style.css (hunk 2 above)
Comment (Um-Brehon) [reply src/components/Menu/style.css]: aboute? Haceeene, voyons* :D
edit: (*trans: come on)

File: src/components/Menu/index.jsx (hunk 3 above)
Comment (BenZen59) [reply src/components/Menu/index.jsx]: div, I don't click on to see the content, I change the classname

File: src/components/Menu/style.css (hunk 2 above)
Comment (LoicBrassart) [reply src/components/Menu/style.css]: @Agent-Roman-Off Bashing/Trolling is allowed (with a modicum of moderation, of course), **aber Französisch ist verbôten ! ರ╭╮ರ**

File: src/components/Menu/index.jsx (hunk 3 above)
Comment (LoicBrassart) [reply src/components/Menu/index.jsx]: To be fair, a **list** would be better as a `<ul><li>` combo anyway

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Change a html tag and we're good

File: src/components/Menu/style.css [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  font-family: "GothamRnd-Book";
+  src:
+    local("GothamRnd-Book"),
+    url("./GothamRnd-Book.otf");
+}
+
+:root {
+  --mColor: #0081c9;
+  --mFont: "GothamRnd-Book";
+}
```
Comment (KeerhemFR) [inline src/components/Menu/style.css]: Will have to remove that ulteriorily, already in global variable

**Review by KeerhemFR — DISMISSED** [review]
Wili have to remind to remove the :root and switch the .css to a style-components.

**Review by KeerhemFR — APPROVED** [review]
See no evil here

**Review by Hacene010 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]

**Pull request merged**